python/cuspatial/cuspatial/tests/binpreds/test_binpred_test_dispatch.py

```diff
@@ -38,10 +23,15 @@ def test_simple_features(
     """Parameterized test fixture that runs a binary predicate test
     for each combination of geometry types and binary predicates.
 
+    Enable the `LOG_DISPATCHED_PREDICATES` environment variable to
+    log the dispatched predicate results.
```

Comment:
The "unhandled" tests are defined between `64: if not LOG_DISPATCHED_PREDICATES:` and `82: else:`. If the variable is not defined, the tests are run normally. Starting on line 83, if `LOG_DISPATCHED_PREDICATES` is defined, the tests are run inside of `try: except:` blocks that handle logging for the errors. It could be refactored in a more sophisticated way, but I think it works as intended. Please give it another look?